tests/visualization/gen_expected_files.py

```diff
@@ -10,26 +11,43 @@
 
 
 def _generate_expected_files(*functions):
+    for (
+        function,
+        show_fn_constants,
+        show_args,
+        link,
```

Comment:
Same value `link` is used for both parameters `link_fn_graphs` and `link_inp_graphs`. Is it voluntary ?

Reply:
Yes, I didn't want to generate an exponential number of tests in the number of configuration parameters, and these seemed similar enough.

Reply:
Ok !